src/backend/distributed/sql/citus--11.0-4--11.1-1.sql

```diff
@@ -178,6 +178,7 @@ CREATE INDEX pg_dist_background_task_depend_depends_on ON pg_catalog.pg_dist_bac
 GRANT SELECT ON pg_catalog.pg_dist_background_task_depend TO PUBLIC;
 
 #include "udfs/citus_job_wait/11.1-1.sql"
+#include "udfs/citus_task_wait/11.1-1.sql"
```

Comment:
This is in the wrong file. It should be in 11.2 migration. Filenames should match that too.